### DISCUSSION TITLE
让芙门永存

### DISCUSSION THREAD

**Discussion by maha258** [share121/inter-knot#388] [其他]
![IMG_20240313_004947_0533](https://github.com/user-attachments/assets/42da6ffb-0e9c-4503-af4e-e0c4ed8c7f1b)
![IMG_20240103_004416_0769](https://github.com/user-attachments/assets/23af63ab-0a1c-414f-84f4-67f8245fae0d)
![IMG_20231215_013418_0850](https://github.com/user-attachments/assets/ee4db882-120a-40f1-a319-4368c845914a)

**Comment by cheeseRatRat** [comment]
最喜欢的一集

**Reply by cheeseRatRat** [reply]
![Uploading 113788104_p0.jpg…]()
